Profiles - ZIB 2017/Extensions and base profiles/extension-valueBoolean.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="extension-valueboolean" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/extension-valueBoolean" />
+  <version value="1.0.0" />
+  <name value="extension-valueBoolean" />
+  <title value="Extension valueBoolean" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A STU3 workaround for Observation.component. The specs ar eincomplete, valueBoolean is not added as a possibility in STU3. It is in R4. With this extension valueBoolean is made possible " />

```

Comment:
Actually, the STU3 spec defines another workaround: " For boolean values use valueCodeableConcept and select codes from <http://hl7.org/fhir/ValueSet/v2-0136> (these "yes/no" concepts can be mapped to the display name "true/false" or other mutually exclusive terms that may be needed")."

Reply:
Used spec workaround